src/Locale/en/Common.json

```
@@ -89,5 +89,19 @@
   "save": "Save",
   "select": "Select",
   "lsg": "Lsg",
-  "delete": "DELETE"
+  "delete": "DELETE",
+  "delete_norm":"Delete",
```

let the uppercase delete be modified by adding `uppercase` class instead, to avoid confusion of usage of keys.

```suggestion
  "delete": "Delete",
```

---

Hey i have done the changes